build.gradle

```diff
@@ -39,11 +53,9 @@ def today = Instant.now()
 def todaysDate = LocalDate.now().format('yyyy-MM-dd')
 version = System.getenv('CI') ? "${todaysDate}.${System.getenv('CIRCLE_BUILD_NUM')}" : todaysDate
 
-jar {
-    baseName = rootProject.name
+bootJar {
```

Comment:
grabbed from elite - this was always incorrect as we aren't using the `jar` task